pkgs/development/libraries/amdvlk/default.nix

```diff
@@ -93,8 +93,8 @@ in stdenv.mkDerivation rec {
     version="$(curl -sL "https://api.github.com/repos/GPUOpen-Drivers/AMDVLK/releases?per_page=1" | jq '.[0].tag_name | split("-") | .[1]' --raw-output)"
     sed -i "pkgs/development/libraries/amdvlk/default.nix" -e 's/version = "[^'"'"'"]*"/version = "'"$version"'"/'
 
-    setHash "$(nix-instantiate --eval -A lib.fakeSha256 | xargs echo)"
-    hash="$(nix to-base64 $(nix-build -A amdvlk 2>&1 | tail -n3 | grep 'got:' | cut -d: -f2- | xargs echo || true))"
+    setHash ""
+    hash="$(nix hash to-base64 $(nix-build -A amdvlk 2>&1 | tail -n3 | grep 'got:' | cut -d: -f2- | xargs echo || true))"
```

Comment:
Does this not need `--experimental-features nix-command` anymore?

Reply:
You’re right. I have this globally in my options, so I didn’t notice. I reverted that part.